From https://docs.python.org/2/tutorial/errors.html#handling-exceptions: &quot;An except clause may name multiple exceptions as a parenthesized tuple, for example&quot;:

    except (IDontLikeYouException, YouAreBeingMeanException) as e:
        pass

Separating the exception from the variable with a comma will still work in Python 2.6 and 2.7, but is now deprecated and does not work in Python 3; now you should be using `as`.